### PR TITLE
add docs about reusable schema references

### PR DESCRIPTION
**Pull request by andrewminion-luminfire**
Partially addresses #14

### CODE REVIEW

**Comment by andrewminion-luminfire** [comment]
Added more examples of tags, request bodies, multiple responses.

File: docs/paths/request-bodies.md [hunk 1]
```diff
@@ -1,5 +1,42 @@
...
+class UserCreateRequestBody extends RequestBodyFactory
+{
+    public function build(): RequestBody
+    {
+        $userSchema = new UserSchema();
+
+        return RequestBody::create('UserCreate')
+            ->description('User data')
+            ->content(
+                MediaType::json()->schema($userSchema->ref())
```
Comment (vyuldashev) [inline docs/paths/request-bodies.md:21]: `UserSchema::ref()` instead of `$userSchema->ref()`

**Comment by vyuldashev** [comment]
Awesome! Thanks!

File: docs/paths/request-bodies.md (hunk 1 above)
Comment (andrewminion-luminfire) [reply docs/paths/request-bodies.md]: Oh, of course…I didn’t look at the `Referencable` trait to notice that.

**Pull request merged**